operator/roles/kiali-deploy/tasks/main.yml

```diff
@@ -46,12 +46,6 @@
   when:
   - kiali_vars.istio_namespace == ""
 
-- name: Set default Grafana namespace
```

Comment:
this section should be resurrected and should build `in_cluster_url` in the same way

Something like:

```
set_fact:
  kiali_vars: "{{ kiali_vars | combine({'external_services': {'grafana': {'in_cluster_url': 'http://grafana.' + kiali_vars.istio_namespace + ':3000'}}}, recursive=True) }}"
when
  kiali_vars.external_services.grafana.auth.in_cluster_url == ""
```

Reply:
of course, change the `name:` to be `Set default Grafana in_cluster_url`

Reply:
all this work is done -- #1145 is merged